lessons/README.md

```diff
@@ -10,6 +10,7 @@ Lesson|.NET Core|.NET 6|Typescript|PowerShell|Python|Agnostic|Contributions by
 |Deployment to Azure|-|-|-|-|[Python](python/http/http-lesson-deploy.md)|[Agnostic](deployment/deployment-lesson.md)|Marc, Dana
 |Cosmos DB Trigger & Bindings|[.NET Core](dotnetcore31/cosmosdb/cosmosdb-lesson-dotnet.md)|-|-|-|-|-|Gabriela, Marc
 |Durable Functions I |-|-|[TypeScript](typescript/durable-functions/chaining/chaining-lesson-ts.md)|-|-|-|Christian, Marc
+|Durable Functions II |-|-|[TypeScript](typescript/durable-functions/advanced/advanced-patterns-lesson.md)|-|-|-|Christian, Marc
```

Comment:
Could you rename the lesson markdown to README.md? Then GH will show it by default when the folder is opened.

Reply:
Will do so

Reply:
Done